src/components/About/About.js

```diff
@@ -47,27 +37,69 @@ class About extends React.Component {
       });
   };
 
+  requestRepolist = (perPage, selectedPage) => {
+    octokit.repos.listForUser({
+      username: 'agatta17',
+      per_page: perPage,
```

Comment:
Codacy found an issue: [Identifier 'per_page' is not in camel case.](https://app.codacy.com/gh/WebHeroSchool/agatta17_todo/pullRequest?prid=6082343)